local_global_letter/experiment.js

```diff
@@ -300,7 +300,10 @@ var start_test_block = {
   data: {
     trial_id: "test_intro"
   },
-  text: '<div class = centerbox><p class = center-block-text>We will now start the test. Press <strong>enter</strong> to begin.</p></div>',
+  text: '<div class = centerbox><p class = center-block-text>We will now start the test. Remember, if the letter is ' +
+    task_colors[0] + ' indicate whether the larger letter is an "H" or "S". If the letter is ' +
+    task_colors[1] +
+    ' indicate whether the smaller letter is an "H" or "S"..</p><p class = center-block-text>Press <strong>enter</strong> to begin.</p></div>',
```

Comment:
I'll fix later. Good catch.

On Saturday, May 28, 2016, Vanessa Sochat <notifications@github.com> wrote:

> In local_global_letter/experiment.js
> <https://github.com/expfactory/expfactory-experiments/pull/459#discussion_r64993306>
> :
>
> > @@ -300,7 +300,10 @@ var start_test_block = {
> >    data: {
> >      trial_id: "test_intro"
> >    },
> > -  text: '<div class = centerbox><p class = center-block-text>We will now start the test. Press <strong>enter</strong> to begin.</p></div>',
> > +  text: '<div class = centerbox><p class = center-block-text>We will now start the test. Remember, if the letter is ' +
> > +    task_colors[0] + ' indicate whether the larger letter is an "H" or "S". If the letter is ' +
> > +    task_colors[1] +
> > +    ' indicate whether the smaller letter is an "H" or "S"..</p><p class = center-block-text>Press <strong>enter</strong> to begin.</p></div>',
>
> two periods / dots here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/expfactory/expfactory-experiments/pull/459/files/441791e16e536560ff810aa21471f622ca426f5f#r64993306>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI5LgIDoxOYim98s-Rn42vEdajHO3Uomks5qGJ6HgaJpZM4IpIwV>
> .
>


-- 
Best,
Ian